Sources/SwiftProtobuf/TextFormatEncodingVisitor.swift

```diff
@@ -59,6 +59,24 @@ internal struct TextFormatEncodingVisitor: Visitor {
     self.encoder = encoder
   }
 
+  private mutating func startField(lookingUp fieldNumber: Int) throws {
+      // This compact form of startField(lookingUp:) slows the entire
```

Comment:
Removed.  (It was there to explain why there were duplicated paths, but I just refactored to eliminate the duplication in a different way.)